cpp/src/neighbors/detail/knn_brute_force.cuh

```diff
@@ -56,9 +56,13 @@
 
 #include <cstdint>
 #include <iostream>
+#include <optional>
 #include <set>
+#include <variant>
 
 namespace cuvs::neighbors::detail {
+
+using namespace cuvs::neighbors::filtering;
```

Comment:
Never use `using namespace` in a header file. It cascades to all header files.

Reply:
All done! Thanks!